packages/foam-vscode/src/core/model/workspace.ts

```diff
@@ -12,23 +11,19 @@ export function getReferenceType(
   if (URI.isUri(reference)) {
```

Comment:
`getReferenceType` may be a method of `URI` I guess. That said, there is (already was) some overloading of the meaning of URI here: a `URI` may be a reference of different types, including `uri`. I think something like `full-uri` would have been a better name for the reference type, but perhaps it's not worth the refactoring.